addons/hw_posbox_homepage/controllers/main.py

```diff
@@ -294,3 +303,28 @@ def enable_ngrok(self, auth_token):
             return 'starting with ' + auth_token
         else:
             return 'already running'
+
+    @http.route('/six_payment_terminal', type='http', auth='none', cors='*', csrf=False)
+    def six_payment_terminal(self):
+        return six_payment_terminal_template.render({
+            'title': 'Six Payment Terminal',
+            'breadcrumb': 'Six Payment Terminal',
+            'terminalId': self.get_six_terminal(),
+        })
+
+    @http.route('/six_payment_terminal_add', type='http', auth='none', cors='*', csrf=False)
+    def add_six_payment_terminal(self, terminal_id):
+        subprocess.check_call("sudo mount -o remount,rw /", shell=True)
+        with open("/home/pi/odoo-six-payment-terminal.conf", "w") as f:
```

Comment:
Be careful and do not use `shell = True` instead uses tagadad
cfr doc: https://docs.python.org/2/library/subprocess.html#frequently-used-arguments
in this case it is not problematic, but take good habits ....